README.md

```diff
@@ -1,167 +1,141 @@
+> **Note for Windows Users:** If using WSL, ensure you have installed WSL2 and Ubuntu. Install Docker Desktop with WSL integration enabled.
 
-If you are using Windows and python was installed using the [python installer](https://www.python.org/downloads/windows/), the option to add python to the path automatically at install time must have been used. If that option was not used and python was manually added to the path, this script will likely not function as expected. Re-run the installer, choose the modify option, check the box to add python to the user path and complete the installation modification prior to the following steps. 
+## Installation
 
-## Install
+### 1. Setup Python Environment
 
-From within the python environment described in the [Environment](#environment) section, run the command:
+We recommend using a virtual environment:
 
-`pip install git+https://github.com/309thEDDGE/opal-release-downloader.git`
+```sh
+python -m venv opal_env
+source opal_env/bin/activate  # On Windows, use `opal_env\Scripts\activate`
+```
 
-_Note: On windows be sure to use_ `python -m pip install ...`.
+### 2. Install OPAL Release Downloader
 
-This command will pull the source from github, then build and install the python package, including dependencies. It will place several scripts on your path in the active environment. Scripts are described in [Download and verify artifacts](#download-and-validate-artifacts).
+```sh
+pip install git+https://github.com/309thEDDGE/opal-release-downloader.git
+```
 
-## Usage
+## Download and Verify OPAL Artifacts
 
-### Runtime arguments: 
+1. **Navigate to an empty directory** where OPAL artifacts will be stored.
+2. **Ensure you are in the correct Python environment.**
+3. **View script options:**
+   ```sh
+   download_opal_artifacts -h
+   ```
+4. **Download artifacts:**
+   ```sh
+   download_opal_artifacts opal-deployment-artifacts
+   ```
+   This downloads and verifies required files into `opal_artifacts/`.
 
-* must be obtained from OPAL developers prior to use
-* S3 bucket name in which OPAL artifacts reside, referred to as `<bucket name>`
-* Release tag of available OPAL artifacts version, in the form of a date _YYYY.MM.DD_, referred to as `<release_tag>`
+## OPAL Deployment
 
-### Download and validate artifacts:
+### 1. Copy Artifacts
 
-* Navigate to an empty directory in which OPAL artifacts shall reside
-* (if applicable) Ensure that the active environment is the one in which opal-release-downloader was installed
-* View the help menu of the download script: `download_opal_artifacts -h`
-* Execute the download script and pass arguments as described in the help menu: `download_opal_artifacts <bucket_name> ...`
-* `<bucket_name>` can be omitted to automatically grab the latest tag
-* See `list_opal_artifacts -h` to view available tags
```

Comment:
Leave the stuff in that you haven't rephrased.

- list_opal_artifacts -h
- If the command runs without error, the `opal_artifacts` directory contains all of the artifacts required to deploy OPAL
- Some of the compressed images are several GBs in size. The download and verification process can take over an hour depending on internet connection and computer performance.
- Do not use the flags `--no-docker` or `--no-rhel` unless you are an expert

etc